documentation/schema.json

```diff
@@ -149,6 +149,17 @@
           "$ref": "#/definitions/TemplateOptions"
         }
       ]
+    },
```

Comment:
Given how verbose adding a new configuration option is to dotnet-monitor, I can split this PR into 2 if folks would prefer that:
- One to add the new exception pipeline step (that doesn't get added)
- One to add this new configuration option